charts/victoria-metrics-agent/README.md

```diff
@@ -1445,7 +1445,7 @@ periodSeconds: 15
       <td>object</td>
       <td><pre class="helm-vars-default-value" language-yaml" lang="plaintext">
 <code class="language-yaml">clusterMode: false
-enabled: false
+enabled: true
```

Comment:
looks like this doc was also updated by accident

Reply:
updated